sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java

```diff
@@ -95,4 +102,36 @@ public static synchronized void init(
       throw new RuntimeException("Failed to initialize Sentry's SDK", e);
     }
   }
+
+  /**
+   * Deduplicate potentially duplicated Fragment and Timber integrations, which can be added
+   * automatically by our SDK as well as by the user. The user's ones win over ours.
+   *
+   * @param options SentryOptions to retrieve integrations from
+   */
+  private static void deduplicateIntegrations(final @NotNull SentryOptions options) {
+    final List<Integration> timberIntegrations = new ArrayList<>();
+    final List<Integration> fragmentIntegrations = new ArrayList<>();
+    for (final Integration integration : options.getIntegrations()) {
```

Comment:
`options.getIntegrations()` is a `CopyOnWriteArrayList`, you can likely use an `Iterator` and remove it directly.

Reply:
But we still need to go through the entire list because we are keeping the latest integrations, I wasn't sure how to make it work with the iterator. Unless we reverse the list

Reply:
Indeed, that's a good point.
If we use an Iterator and traverse using a reverse other, 2 booleans would do the job, but it's a nit, it'd not make much difference anyway, your call.

Reply:
decided to keep it as is for now.